dms.py

```diff
@@ -87,18 +90,18 @@ def listofdir(func, filter_to):
     return "".join(rv)
 
 
-def link_list(es):
-    rv = ["<ul style='display: block; padding: 1em; '>"]
-    for e in es:
-        rv.append("<li>")
-        rv.append(e)
-        rv.append("</li>")
-    rv.append("</ul>")
-    return "".join(rv)
+def link_list(items, attributes = 'class="list_items"' ):
+    ulist = [ ("<ul %s>" % attributes) ]
```

Comment:
Wrong usage of spaces and parens: please remove parens here and also the spaces around those.